e2e-tests/stream-mode-tests.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+
+#######################################
+# Function that prints messages
+# Arguments:
+#   anything that needs printing.
+#######################################
+function print_message() {
+  local print_prefix="E2E: STREAM MODE TEST:"
+  echo "${print_prefix} $*"
+}
+
+#######################################
+# Call common-mode-tests to setup 
+# the testing env
+#  
+#######################################
+source ./e2e-tests/common-mode-tests.sh
+setup
+
+print_message "OPENMRS AND FHIR SERVER STARTED SUCCESSFULLY"
+
+test_dir=$(mktemp -d -t analytics_tests__XXXXXX)
+
+print_message "PARQUET FILES WILL BE WRITTEN INTO $test_dir DIRECTORY"
+
+print_message "BUILDING THE ANALYTICS  PIPELINE"
+
+timeout 1m mvn compile exec:java -pl streaming-binlog \
+   -Dexec.args="--openmrsUserName=admin --openmrsPassword=Admin123 \
+   --openmrsServerUrl=http://localhost:8099/openmrs \
+   --openmrsfhirBaseEndpoint=/ws/fhir2/R4 \
+   --outputParquetPath='${test_dir}' --fhirSinkPath=http://localhost:8098/fhir --sinkUserName=hapi --sinkPassword=hai "
+
+
+print_message "SEARCH INDEX REBUILD"
+
+curl -i -u admin:Admin123 -X POST http://localhost:8099/openmrs/ws/rest/v1/searchindexupdate
+
+print_message "CREATE A PATIENT IN OPENMRS TO BE STREAMED"
+
+curl -i -u admin:Admin123 -H "Content-type: application/json" -X POST http://localhost:8099/openmrs/ws/fhir2/R4/Patient -d @./utils/patient.json
+
+print_message "CREATE AN ENCOUNTER IN OPENMRS TO BE STREAMED"
+
+curl -i -u admin:Admin123 -H "Content-type: application/json" -X POST http://localhost:8099/openmrs/ws/rest/v1/encounter -d @./utils/encounter.json
+
+print_message "CREATE AN OBSERVATION IN OPENMRS TO BE STREAMED"
+
+curl -i -u admin:Admin123 -H "Content-type: application/json" -X POST http://localhost:8099/openmrs/ws/fhir2/R4/Observation -d @./utils/obs.json
+
+ print_message "RUNNING STREAM MODE WITH PARQUET & FHIR SYNC"
+ print_message "mvn compile exec:java -pl streaming-binlog \
```

Comment:
Instead of repeating the `mvn` command, please wrap it into a variable and use it for this print and next command.